docs/tutorials/r/classifyRealImageWithPretrainedModel.md

```diff
@@ -89,7 +89,7 @@ Load and plot the image:
     plot(im)
  ```
 
-![plot of chunk unnamed-chunk-5](../../web-data/mxnet/knitr/classifyRealImageWithPretrainedModel-unnamed-chunk-5-1.png)
+![plot of chunk unnamed-chunk-5](.https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/knitr/classifyRealImageWithPretrainedModel-unnamed-chunk-5-1.png)
```

Comment:
Extra "." before the URL?

Reply:
Addressed the period before the link. Thanks!